db/migrate/20230809092310_add_user_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToPost < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :posts, :user, foreign_key: true
+  end
```

Comment:
Added in the same migration that we use for creating post table

Reply:
Done